gradle/repositories.gradle.kts

```diff
@@ -9,3 +9,12 @@ listOf(pluginManagement.repositories, dependencyResolutionManagement.repositorie
     maven("https://storage.googleapis.com/gradleup/m2")
   }
 }
+
+pluginManagement.repositories {
+  exclusiveContent {
+    forRepository { maven("https://storage.googleapis.com/gradleup/m2") }
+    filter {
+      includeGroup("com.gradleup.librarian")
+    }
+  }
+}
```

Comment:
I know we have this in other places but this is actually wrong. `librarian` is also released on Maven Central. 

What we need is more the opposite:

```kotlin

  maven("https://storage.googleapis.com/gradleup/m2") {
    content {
      includeGroup("com.gradleup.librarian")
    }
  }

```